packages/wdio-junit-reporter/src/types.ts

```diff
@@ -13,12 +15,11 @@ interface ClassNameFormatOptions {
     /**
      * Context of the current suite
      */
-    suite?: SuiteStats,
+    suite?: SuiteStats
```

Comment:
`suite` seems to be always given, so let's improve this.

```suggestion
    suite: SuiteStats
```